packages/react-switch/src/components/Switch/useSwitch.ts

```diff
@@ -1,25 +1,41 @@
 import * as React from 'react';
+import { getNativeElementProps, useId, resolveShorthand } from '@fluentui/react-utilities';
```

Comment:
nit: Order these alphabetically

```suggestion
import { getNativeElementProps, resolveShorthand, useId } from '@fluentui/react-utilities';
```